apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/wpcom-nux.js

```diff
@@ -111,6 +111,8 @@ function WpcomNux() {
 	);
 }
 
-registerPlugin( 'wpcom-block-editor-nux', {
-	render: () => <WpcomNux />,
-} );
+if ( __experimentalCreateInterpolateElement && Guide && GuidePage ) {
```

Comment:
I think we should add a comment explaining why this is needed.

Reply:
added